packages/backend-tasks/src/tasks/PluginTaskSchedulerJanitor.ts

```diff
@@ -53,15 +53,31 @@ export class PluginTaskSchedulerJanitor {
 
   private async runOnce() {
     const dbNull = this.knex.raw('null');
+    let tasks = [];
```

Comment:
Can you make sure to put an explicit type on this just to make sure that both branches return the right thing and that readers know upfront what's going in here? Like

```ts
let tasks: Array<{ id: string }>;
```

should be possible to not assign `[]` to it too, I think, the compiler should make sure that all branches assign to it properly before reading.